dockers/docker-fpm-frr/frr.conf.j2

```diff
@@ -79,6 +79,62 @@ route-map FROM_BGP_SPEAKER_V4 permit 10
 !
 route-map TO_BGP_SPEAKER_V4 deny 10
 !
+{% block vnet_bgp_instance %}
+{% if VNET is defined %}
```

Comment:
where does this VNET comes from?

Reply:
https://github.com/Azure/SONiC/blob/master/doc/vxlan/Vxlan_hld.md#211-vxlan-table
VNET comes from the ConfigDB